DeceptionGame/Assets/Scripts/MapGeneration/MeshGenerator.cs

```diff
@@ -0,0 +1,425 @@
+using UnityEngine;
+using System.Collections;
```

Comment:
This is where the black magic happens. It just converts the map into a mesh. I don't think you will ever need to modify this script.